### ISSUE TITLE
Feature: Add Mute Button

### ISSUE THREAD

**Issue by nikhilk0** [xbmc/Official-Kodi-Remote-iOS#269]
Any chance of mute button getting added to the app? This feature would be pretty helpful while watching Live TV to mute commercials. It would be better to mute than decrease the volume by the step of 2.

**Comment by wutschel** [comment]
I found this a useful request and worked on this. Let me share a few screenshots from an intermediate state of play. I am still working on the logic and need to use own icons instead of the system icons, but the screenshots give an idea of how this looks like and will work. Screenshots always show (top to bottom): before, new muted, new unmuted.

iPad:
On iPad I am adding a mute button left to the button which reduces volume. The same "metal button" background is used. When mute is active the foreground icon is becoming red. Unmute can be done via either pressing the mute button or one of volume up/down buttons.
<a href="https://abload.de/image.php?img=bildschirmfoto2021-05yjkih.png"><img src="https://abload.de/img/bildschirmfoto2021-05yjkih.png" /></a>

iPhone:
On iPhone I am adding a mute button left to volume slider. Instead of supporting buttons, I now only kept the slider and icons at left/right of the slider. When mute is active the foreground icon is becoming red and the slider is greyed out. It will only become usable when pressing the mute button to unmute. I personally like the new design.
<a href="https://abload.de/image.php?img=bildschirmfoto2021-05dzj8r.png"><img src="https://abload.de/img/bildschirmfoto2021-05dzj8r.png" /></a>
